glossar.tex

```diff
@@ -32,14 +32,38 @@
 
 \newglossaryentry{AM}{
 	name={AM},
-	first={Angewandte Mathematik (AM, INF 294)},
-	description={In der Angewandten Mathematik befinden sich die Bereichsbibliothek sowie ein Kopierer, an dem Mathe- und Infostudis aus Studiengebühren finanziert kopieren können. Die Kopierkarte kann bei der Bibliotheksaufsicht gegen Studi- und Personalausweis ausgeliehen werden. Unseres Wissens nach ist dies auch das einzige Gebäude mit negativen Raumnummern. Wenn ihr also eine Übungsgruppe in „-104“ habt, ist der Keller der AM gemeint}
+	first={Angewandte Mathematik},
+	description={Die Angewandte Mathematik ist eines der Institute der Fakultät für Mathematik und Informatik und sitzt im Mathematikon}
+}
+
+\newglossaryentry{IAM}{
+	name={IAM},
+	first={Institut für Angewandte Mathematik},
+	description={siehe \Gls{AM}}
+}
+
+\newglossaryentry{MI}{
+	name={MI},
+	first={Mathematisches Institut},
+	description={Das Mathematisches Institut (auch Reine Mathematik genannt) ist eines der Institute der Fakultät für Mathematik und Informatik und sitzt im Mathematikon im 3. OG}
 }
 
 \newglossaryentry{RM}{
 	name={RM},
-	first={Reine Mathematik (RM, INF 288)},
-	description={Die Reine Mathematik ist das Gebäude INF 288. Im ersten Stock befinden sich die Prüfungssekretariate der Mathe}
+	first={Reine Mathematik},
+	description={siehe \Gls{MI}}
+}
+
+\newglossaryentry{IfI}{
+	name={IfI},
+	first={Institut für Informatik},
+	description={Das Institut für Informatik ist eines der Institute der Fakultät für Mathematik und Informatik und sitzt im Mathematikon. Die Mitglieder decken unter anderem ein Großteil der Lehre im Bereich Informatik ab}
+}
+
+\newglossaryentry{Mathematikon}{
+    name={Mathematikon},
+	first={Mathematikon},
+	description={Das Mathematikon (INF 205) ist das neu gebaute Gebäude an der Berliner Straße in dem die Fakultät für Mathematik und Informatik und das IWR untergebracht sind. Dort finden auch die meisten Seminare, Übungen und kleinere Vorlesungen statt. Außerdem befindet sich dort die Bereichsbibliothek, einige Arbeitsplätze und natürlich die Fachschaft}
```

Comment:
In #54 wurde auch der Glossareintrag Mathematikon eingefügt, jedoch nicht mit so viel Informationen. Darauf sollte bei einem merge geachtet werden, dass nicht beide dann auftauchen.